cg/meta/orders/metagenome_submitter.py

```diff
@@ -52,18 +70,26 @@ def order_to_status(order: OrderIn) -> dict:
         return status_data
 
     def store_items_in_status(
-        self, customer: str, order: str, ordered: dt.datetime, ticket: int, items: List[dict]
+        self,
+        customer: str,
+        data_analysis: Pipeline,
```

Comment:
Here we change the signature of a method that we override, it would be nice to discuss the implications of that and what way we should go, if any

Reply:
Agreed I noticed that, but since it was done in other submitters I let it slide. But let's discuss it 👍

Reply:
discussed, and agreed to generalize